src/azure-cli/azure/cli/command_modules/acs/azuremonitormetrics/recordingrules/create.py

```diff
@@ -50,23 +51,30 @@ def put_rules(cmd, default_rule_group_id, default_rule_group_name, mac_region, a
 
 # pylint: disable=line-too-long
 def create_rules(cmd, cluster_subscription, cluster_resource_group_name, cluster_name, azure_monitor_workspace_resource_id, mac_region, raw_parameters):
+    # limit rule group name to 260 characters
     # with urllib.request.urlopen("https://defaultrulessc.blob.core.windows.net/defaultrules/ManagedPrometheusDefaultRecordingRules.json") as url:
     #     default_rules_template = json.loads(url.read().decode())
     default_rules_template = get_recording_rules_template(cmd, azure_monitor_workspace_resource_id)
-    default_rule_group_name = "NodeRecordingRulesRuleGroup-{0}".format(cluster_name)
+    default_rule_group_name = "NodeRecordingRulesRuleGroup-{0}".format(cluster_name[:232])
```

Comment:
use a function to truncate the concatenated name to a maximum of 260 characters would be more consistent in my opinion

Reply:
https://github.com/Azure/azure-cli/pull/27788/files